source/linux/Foundational_Components/Kernel/Kernel_Drivers/Network/HSR_Offload.rst

```diff
@@ -98,19 +90,11 @@ forwarding and Tx packet duplication offloaded
           ethtool -K "$ifb" hsr-tag-ins-offload on
           ethtool -K "$ifb" hsr-tag-rm-offload on
           ethtool -k "$ifb" | grep hsr
-
-          devlink dev param set "$device" name hsr_offload_mode value true cmode runtime
   fi
```

Comment:
@danish-ti, while unrelated to this PR for cleanup, maybe an else condition is required to disable offload when "hsr_sw" is requested. I am approving this PR since the cleanup looks good to me.

Reply:
I see. Seems like the script assumes that all the offload flags are `off` by default and doesn't do anything when `hsr_sw` mode is selected.

@Siddharth-Vadapalli-at-TI Thanks for approving this. I will create another PR do the cleanup once this gets merged.

@cshilwant  Pls merge this PR.